src/page/index.ts

```diff
@@ -25,69 +27,95 @@ const cooldown = getEnvVariable('COOLDOWN', envVars) as number;
+const recaptchaSiteKey = getEnvVariable('GOOGLE_CAPTCHA_SITE_KEY', envVars);
 
-    if (isHuman) {
-      try {
-        decodeAddress(address);
-      } catch (e) {
-        res.status(400).send('Unparsable address.');
-        return;
-      }
+function sendResponse(
+  isHuman: boolean,
+  address: string | Uint8Array | null | undefined,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  response: Response<any, any>
+) {
+  if (isHuman) {
+    try {
+      decodeAddress(address);
+    } catch (e) {
+      response.status(400).send('Unparsable address.');
+      return;
+    }
 
-      const attemptTime = Date.now();
+    const attemptTime = Date.now();
 
-      if (attemptTime - cooldown > lastSuccess) {
-        ax.post<DripResponse>('/page-endpoint', {
-          address,
-          amount: dripAmount,
-        })
-          .then((inner_res) => {
-            if (isDripSuccessResponse(inner_res.data)) {
-              res
-                .status(201)
-                .send(`{"message": "Sent ${dripAmount} ${unit}s."}`);
-              lastSuccess = attemptTime;
-            } else {
-              res
-                .status(429)
-                .send(
-                  inner_res.data.error ||
-                    'An unexpected error occured, please contact us and complain'
-                );
-              return;
-            }
-          })
-          .catch((e) => {
-            res
-              .status(500)
+    if (attemptTime - cooldown > lastSuccess) {
+      ax.post<DripResponse>('/page-endpoint', {
+        address,
+        amount: dripAmount,
+      })
+        .then((inner_res) => {
+          if (isDripSuccessResponse(inner_res.data)) {
+            response
+              .status(201)
+              .send(`{"message": "Sent ${dripAmount} ${unit}s."}`);
+            lastSuccess = attemptTime;
+          } else {
+            response
+              .status(429)
               .send(
-                'An unexpected error occured, please contact us and complain'
+                inner_res.data.error ||
+                  'An unexpected error occurred, please contact us and complain'
               );
-          });
-      } else {
-        res.status(503).send('Too many faucet requests.');
-      }
+            return;
+          }
+        })
+        .catch(() => {
+          response
+            .status(500)
+            .send(
+              'An unexpected error occurred, please contact us and complain'
+            );
+        });
     } else {
-      res.status(401).send('Google captcha failed.');
-      return;
+      response.status(503).send('Too many faucet requests.');
     }
+  } else {
+    response.status(401).send('Google captcha failed.');
+    return;
+  }
+}
+
+app.post('/drip', (req, userResponse) => {
+  const address = req.body.address;
+
+  if (recaptchaSecretKey !== undefined) {
+    console.log(recaptchaSecretKey);
```

Comment:
Suggestion: remove this, I assume this is left by accident?

Reply:
Good point, removed